### ISSUE TITLE
Swift registry support

### ISSUE THREAD

**Issue by martinm82** [jfrog/terraform-provider-artifactory#489]
**Is your feature request related to a problem? Please describe.**
With Artifactory 7.39.4 Swift registry support was introduced. It would be great cover this feature as well in this provider.

**Comment by chb0github** [comment]
See what we can do - we aim to please

**Comment by chb0github** [comment]
@alexhung I am looking at one of our own instances, as well as a bleeding edge container and I see no swift support. @martinm82 can you actually create one, or did you read a press release and assume it was ready?
```
curl https://partnership.jfrog.io/artifactory/api/system/version | jq -re .version
7.41.1
```

But I see see the swift addon installed
```
curl https://partnership.jfrog.io/artifactory/api/system/version | jq -re '.addons | map(select(. == "swift")) | first'
swift
```

This [super recent announcement](https://jfrog.com/blog/artifactory-your-swift-package-repository/) says [it's available](https://www.jfrog.com/confluence/display/JFROG/Swift+Registry#SwiftRegistry-SettingupaLocalRepository) but I'm not seeing it
![Screen Shot 2022-07-01 at 8 54 40 PM](https://user-images.githubusercontent.com/1476816/176985607-7ad4f0d1-5c0a-4e10-9560-9bc5d6280c56.png)

**Comment by martinmigasiewicz-tomtom** [comment]
I am on mobile phone only but we have the Swift support available:

![image](https://user-images.githubusercontent.com/71527391/176989635-7f48dc21-b1cb-49cd-a725-5ff4d58227ca.jpeg)

We are on E+ Cloud 7.39.6.

**Comment by chb0github** [comment]
Can you give me the output of
`curl ${yourhost}/artifactory/api/system/version `

**Comment by chb0github** [comment]
Ok, turns out it's a UI bug that will be fixed in `7.41.3` and in fact, it is supported (hence my confusion about it being in the addons list)
